frontend/src/component/project/ProjectAccess/ProjectAccessTable/ProjectAccessTable.tsx

```diff
@@ -151,11 +151,13 @@ export const ProjectAccessTable: VFC = () => {
                 id: 'role',
                 Header: 'Role',
                 accessor: (row: IProjectAccess) =>
-                    row.entity.roles.length > 1
-                        ? `${row.entity.roles.length} roles`
-                        : access?.roles.find(
-                              ({ id }) => id === row.entity.roleId
-                          )?.name,
+                    row.entity.roles
+                        ? row.entity.roles.length > 1
+                            ? `${row.entity.roles.length} roles`
+                            : access?.roles.find(
+                                  ({ id }) => id === row.entity.roleId
+                              )?.name
+                        : 'No Roles!',
```

Comment:
@gastonfournier I think this changed slipped by me and I'm curious: What was the reason for it? Do we expect any reason why `roles` can be falsy?

Reply:
Oh, I think this was accidental when trying to figure out why the tests were failing, I added this to try to see if this was the issue in cypress